src/modules/account/api/upload-image.api.ts

```diff
@@ -0,0 +1,16 @@
+import { apiClient } from '#/shared/libs/api-client.lib';
+import { useMutation } from '@tanstack/react-query';
+
+interface UploadResponse {
+  url: string;
+}
+
+export function useUploadImageMutation() {
+  const mutation = useMutation<UploadResponse, Error, FormData>({
+    mutationFn: async (formdata: FormData) => {
+      const response = await apiClient.post<UploadResponse>('/upload/image', formdata);
```

Comment:
Is this endpoint not returning the response in BaseResponse format?

Reply:
yes, it isn't

Reply:
> yes, it isn't

If so, could you please fix it in BE? We can't have one endpoint return data in a different format than our convention. Please modify this to accept BaseResponse as well.